common/rabbitmq/templates/certificate.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.enableSsl }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: tls-{{ template "fullname" . }}
+  labels:
+    {{- include "rabbitmq.labels" (list $ "version" "rabbitmq" "deployment" "messagequeue") | indent 4 }}
+spec:
+  secretName: tls-{{ template "fullname" . }}
+  secretTemplate:
+    labels:
+      {{- include "rabbitmq.labels" (list $ "version" "rabbitmq" "deployment" "messagequeue") | indent 6 }}
+  commonName: "{{ template "fullname" . }}.{{ .Release.Namespace }}.svc.kubernetes.{{ .Values.global.region | required "global.region missing" }}.{{ .Values.global.tld | required "global.tld missing" }}"
+  dnsNames:
+    - "{{ template "fullname" . }}.{{ .Release.Namespace }}.svc.kubernetes.{{ .Values.global.region | required "global.region missing" }}.{{ .Values.global.tld | required "global.tld missing" }}"
+  {{- if .Values.externalNames }}
+    {{- range .Values.externalNames }}
+    - "{{ . }}"
+    {{- end }}
+  {{- end }}
```

Comment:
`externalNames` doesn't seem to be a documented feature of the `rabbitmq` chart. it's not used anywhere else before this commit and not part of the changelog.

Reply:
True, I've added a comment to the changelog.